conda/activate.py

```diff
@@ -147,6 +157,10 @@ def deactivate(self):
         return self._finalize(self._yield_commands(self.build_deactivate()),
                               self.tempfile_extension)
 
+    def deactivatepost(self):
+        return self._finalize(self._yield_commands(self.build_deactivatepost()),
+                              self.tempfile_extension)
+
     def reactivate(self):
```

Comment:
For `reactivate` we also need a two-staged process if we have `deactivate` scripts.